### DISCUSSION TITLE
PG extension updates not available to existing projects

### DISCUSSION THREAD

**Discussion by yan-hic** [supabase/supabase#13425] [Questions] [answered]
It seems existing projects cannot benefit from extension updates. 
See https://github.com/supabase/wrappers/pull/74 for context. I had raised a few issues/requests for the bigquery wrapper. The team was quick to respond and has merged the changes, but I can't enable the updated 0.1.9 version. 

Starting over a new project as suggested just for the sake of an extension update is really an overkill. Can't believe there is no simple other way - or was that an oversight. 
Would it be different with a paid plan ?

**Comment by GaryAustin1** [comment, marked as answer]
If you have a free plan you can pause/restore your project to get the latest code.

**Reply by yan-hic** [reply]
Yeah ! Thanks @GaryAustin1 
It would be good to mention the how-to in the documentation and/or on the Dashboard.

**Comment by pjmv** [comment]
The accepted answer should be taken with a grain of salt : I had to go that route for a similar reason and doing so broke my install. See https://github.com/supabase/wrappers/issues/131.

The core of the issue seems to be an authorization problem, and because supabase users do not have access to superuser users in their postgres instance I am not sure what I can do beyond creating another project and duplicating everything.

**Reply by GaryAustin1** [reply]
You should contact support and see if they can help.

**Reply by pjmv** [reply]
Hi Gary, Thanks again for the outstanding support ! I "solved" my problem by nuking everything and reinstalling everything. Not a big deal for me and surely I fell in a niche state.
